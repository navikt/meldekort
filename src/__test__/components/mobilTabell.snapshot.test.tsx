import * as React from "react";
import * as renderer from "react-test-renderer";
import ProviderWrapper from "../testSetup/providerWrapper";
import MobilTabell from "../../app/components/tabell/mobil/mobilTabell";

it("MobilTabell", () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <MobilTabell
        columns={[
          {
            key: "test",
            label: <span>TEST</span>,
          },
        ]}
      />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
