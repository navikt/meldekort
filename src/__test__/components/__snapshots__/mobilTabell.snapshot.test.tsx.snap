// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobilTabell 1`] = `
<div
  className="mobilTabell-container"
/>
`;
